ocs_ci/deployment/ibm.py

```diff
@@ -0,0 +1,548 @@
+                validate_pvc_are_mounted_on_monitoring_pods
+            )(pods_list)
+        elif config.ENV_DATA.get('monitoring_enabled') and config.ENV_DATA.get("telemeter_server_url"):
+            # Create configmap cluster-monitoring-config to reconfigure
+            # telemeter server url when 'persistent-monitoring' is False
+            create_configmap_cluster_monitoring_pod(
+                telemeter_server_url=config.ENV_DATA["telemeter_server_url"])
+
+        # Change registry backend to OCS CEPHFS RWX PVC
+        registry.change_registry_backend_to_ocs()
+
+        # Verify health of ceph cluster
+        # TODO: move destroy cluster logic to new CLI usage pattern?
+        logger.info("Done creating rook resources, waiting for HEALTH_OK")
+        try:
+            ceph_health_check(namespace=self.namespace, tries=30, delay=10)
+        except CephHealthException as ex:
+            err = str(ex)
+            logger.warning(f"Ceph health check failed with {err}")
+            if "clock skew detected" in err:
+                logger.info(
+                    f"Changing NTP on compute nodes to"
+                    f" {constants.RH_NTP_CLOCK}"
+                )
+                update_ntp_compute_nodes()
+                assert ceph_health_check(
+                    namespace=self.namespace, tries=60, delay=10
+                )
+
+        # patch gp2/thin storage class as 'non-default'
+        self.patch_default_sc_to_non_default()
+
+        # Modify Noobaa endpoint auto scale values according to the cluster specs
+        # if check_nodes_specs(min_cpu=constants.MIN_NODE_CPU, min_memory=constants.MIN_NODE_MEMORY):
+        #     logger.info(
+        #         "The cluster specs meet the minimum requirements and "
+        #         "therefore, NooBaa auto scale will be enabled"
+        #     )
+        #     min_nb_eps = config.DEPLOYMENT.get('min_noobaa_endpoints')
+        #     max_nb_eps = config.DEPLOYMENT.get('max_noobaa_endpoints')
+        #     change_noobaa_endpoints_count(min_nb_eps=min_nb_eps, max_nb_eps=max_nb_eps)
+        # else:
+        #     logger.warning(
+        #         "The cluster specs do not meet the minimum requirements and "
+        #         "therefore, NooBaa auto scale will remain with its default values"
+        #     )
+        #     min_eps = 1
+        #     max_eps = 1 if float(config.ENV_DATA['ocs_version']) < 4.6 else 2
+        #     logger.info(
+        #         f"The Noobaa endpoint auto scale values: min: {min_eps}, max: {max_eps}"
+        #     )
+
+    def destroy_cluster(self, log_level="DEBUG"):
+        """
+        Base destroy cluster method, for more platform specific stuff please
+        overload this method in child class.
+
+        Args:
+            log_level (str): log level for installer (default: DEBUG)
+        """
+        if not config.ENV_DATA['skip_ocs_deployment']:
+            self.destroy_ocs()
```

Comment:
Is this necessary since by destroying OCP we inherently destroy OCS as well?

Reply:
I want to cover the case of destroying OCS only and not OCP.